cds/base/dojson/marc21/translations/default.py

```diff
@@ -17,14 +17,16 @@
 # along with Invenio; if not, write to the Free Software Foundation, Inc.,
 # 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
 
-from dojson.overdo import Overdo
 from dojson.contrib.marc21 import marc21
+from dojson.overdo import Overdo
 
 
 class CDSMarc21(Overdo):
 
     """Translation Index for CDS specific MARC21."""
 
+    __query__ = ''
```

Comment:
Probably we should add here a query for all record that we want to match with this one, i.e `690C_.a:CERN`

I know that we are using this one also for videos, either we add a new dummy class for them inheriting from this and we put a different query:
```python
class Video(CDSMarc21):
    __query__ = '980__.a:Video'
translation = Video()
```
or we add everything we want to match to this query with `OR`

I like better the first option :wink: